### PR TITLE
12.0 moka

### PR DESCRIPTION
**Pull request by gregory-moka**
Edit app to totally hide column on report (SO and bill) and on web views.

### CODE REVIEW

File: sale_report_hide_price/models/__init__.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
-# See LICENSE file for full copyright and licensing details.
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd. <sebastien.beau@akretion.com>
```
Comment (JayVora-SerpentCS) [inline sale_report_hide_price/models/__init__.py:2]: Sorry, this mismatches

File: sale_report_hide_price/models/__init__.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from . import sale
\ No newline at end of file
+from . import sale
+from . import bill
```
Comment (JayVora-SerpentCS) [inline sale_report_hide_price/models/__init__.py]: can you rename it as invoice.py ? makes more sense.

File: sale_report_hide_price/models/bill.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd. <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api
+from openerp import fields, models
+
+class BillOrder(models.Model):
```
Comment (JayVora-SerpentCS) [inline sale_report_hide_price/models/bill.py]: You should make new module if its invoice report hiding it.
The name of the module should match the purpose.
We would appreciate a new module if you make.

Comment (gregory-moka) [reply sale_report_hide_price/models/bill.py]: Sure, but, if our customers wants to hide price on sale orders, they want to hide this prices on invoice too. For them, it's obvious to hide price on invoice too.
Maybe the module name can be changed too ?

**Comment by JayVora-SerpentCS** [comment]
Can you fix conflicts plz?

**Pull request closed without merging**